elements/pfe-accordion/src/pfe-accordion.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
@chrisdo1 Would you mind taking a look at this schema and the updates I made to the file above (pfe-accordion.js) to ensure they match what you know about the component.  I didn't see any styles or use of `color` or `theme` so I removed those from the component.

Reply:
@castastrophe Thanks for the heads up. This looks correct although it does contain the theme settings (on="dark") which you have accounted for in the schema.